lib/dataBroker.js

```diff
@@ -56,7 +56,7 @@ module.exports = function(streamDAO) {
             if (!schema.isValidVersion(versionNum, uploadConfig.minimumUploaderVersion)) {
               return cb({
                 statusCode: 400,
-                text: 'The minimum supported version is ['+uploadConfig.minimumUploaderVersion+']. Version ['+datum.version+'] is no longer supported.',
+                message: 'The minimum supported version is ['+uploadConfig.minimumUploaderVersion+']. Version ['+datum.version+'] is no longer supported.',
```

Comment:
Perhaps this change is what's causing the uploader to fail to redirect to the "you need to update your uploader" when the version mismatch happens? Dunno, but the advantage of keeping it the same could be that versions of the uploader that are _already_ out in the wild will also benefit from the fix(es).

Maybe it's worth testing the prod uploader against new jellyfish under version-mismatch conditions?